### DISCUSSION TITLE
Lint Job Debugging tools

### DISCUSSION THREAD

**Discussion by gravitinos** [trunk-io/docs#321] [Feature Requests]
I'd like a way to take my slow trunk check, and narrow down which lint jobs are taking the longest. Ideally I could dive deeper into each lint job too if available, to see what parts of each process is taking the longest.

I'd also be happy for a guide on how to do this if it's not a feature.

This aligns with my ultimate goal of having my code editor feel as snappy as possible, and sometimes trunk jobs seem to get hung up because it takes so long.

**Comment by gewenyu99** [comment]
Hey, thank you for raising this request. I do think some way of reporting and profiling linter performance would be interesting. Linking this to the team to discuss. There might already be logging that does similar things, but I do thing a command line option when you run check would be cool, too!
